macos/JackTrip.app_template/Contents/Info.plist

```diff
@@ -11,11 +11,11 @@
 	<key>CFBundleIconFile</key>
 	<string>jacktrip</string>
 	<key>CFBundleIdentifier</key>
-	<string>org.psi-borg.qjacktrip</string>
+	<string>org.psi-borg.JackTrip</string>
```

Comment:
It might be worth leaving the bundle identifier as is. It's not user facing, and on the installer/upgrade front, we may be able to leverage it to guard against parallel installation issues. Either way, an all lowercase bundle ID is generally considered best practice. (There are some weird case issues that can occur otherwise.)

Reply:
Also, have just created an updated bundle script in the macos-assemble-script branch. It takes a version number as an argument so that that gets consistently set in both the bundle and the package, and you can optionally specify the app name and bundle identifier. (If you don't it reverts to defaults, which are currently my defaults, but could easily be changed.)

Reply:
(So, whatever gets decided on the naming front, it's a simple variable change. At least on the macos front.)

Reply:
I'll take a look, thanks!